client/landing/gutenboarding/components/header/style.scss

```diff
@@ -58,22 +58,6 @@ $padding--gutenboarding__header-domain-picker-button: 0.3em 0.5em;
 
 	&:disabled {
 		opacity: 1;
-		.gutenboarding__site-title {
-			color: black;
-		}
-	}
-
-	.gutenboarding__site-title {
-		padding: 0;
-	}
-
-	> span {
```

Comment:
Need to double check if this is any longer needed.

Reply:
The `span` is used here: https://github.com/Automattic/wp-calypso/blob/update/gutenboarding-header-styles/client/landing/gutenboarding/components/domain-picker-button/index.tsx#L79

But I can't tell what's the harm of removing it.

Reply:
I can't tell either. Let's just remove.